docs.json

```diff
@@ -87,15 +87,22 @@
           {
             "group": "Configurations",
             "pages": [
+              {
+                "group": "Add-Ons",
+                "icon": "puzzle-piece",
+                "pages": [
+                  "settings/add-ons/feedback",
+                  "settings/add-ons/ci",
+                  "settings/add-ons/preview-deployments"
+                ]
+              },
```

Comment:
Can we add these to the bottom? They shouldn't be above custom domain or SEO amongst others

Reply:
Done